pattern_importer.py

```
@@ -0,0 +1,322 @@
+                
+                # Extract regex (handle both ''' and "" formats)
+                regex_match = re.search(r'regex\s*=\s*\'\'\'([^\']+)\'\'\'', section, re.DOTALL)
+                if not regex_match:
+                    regex_match = re.search(r'regex\s*=\s*"([^"]+)"', section)
+                
+                if regex_match:
+                    regex_pattern = regex_match.group(1).strip()
+                    patterns[rule_id] = regex_pattern
+            
+            logger.info(f"Loaded {len(patterns)} patterns from GitLeaks")
+            return patterns
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)